src/services/blocks/BlocksService.ts

```diff
@@ -72,9 +68,11 @@ export class BlocksService extends AbstractService {
 			digest,
 		} = block.header;
 
-		const logs = digest.logs.map((log) => {
-			const { type, index, value } = log;
+		const authorId = sessionValidators
+			? this.extractAuthor(sessionValidators, digest)
+			: undefined;
 
+		const logs = digest.logs.map(({ type, index, value }) => {
```

Comment:
is this `.map` doing anything?

Reply:
Good question - I didn't dig into who wrote it / why, but it looks like it just pulls out some select fields. Not sure why it was done.

Reply:
Yea so I did some further digging, its just refactoring the log data, and leaving out the fluff. 

Example: (Querying head from a local dev node)
``` typescript
// BEFORE
Vec(2) [
  Type {
    registry: TypeRegistry {},
    isOther: [Getter],
    asOther: [Getter],
    isAuthoritiesChange: [Getter],
    asAuthoritiesChange: [Getter],
    isChangesTrieRoot: [Getter],
    asChangesTrieRoot: [Getter],
    isSealV0: [Getter],
    asSealV0: [Getter],
    isConsensus: [Getter],
    asConsensus: [Getter],
    isSeal: [Getter],
    asSeal: [Getter],
    isPreRuntime: [Getter],
    asPreRuntime: [Getter]
  },
  Type {
    registry: TypeRegistry {},
    isOther: [Getter],
    asOther: [Getter],
    isAuthoritiesChange: [Getter],
    asAuthoritiesChange: [Getter],
    isChangesTrieRoot: [Getter],
    asChangesTrieRoot: [Getter],
    isSealV0: [Getter],
    asSealV0: [Getter],
    isConsensus: [Getter],
    asConsensus: [Getter],
    isSeal: [Getter],
    asSeal: [Getter],
    isPreRuntime: [Getter],
    asPreRuntime: [Getter]
  },
  registry: TypeRegistry {},
  _Type: [class Type]
]


// AFTER

[
  {
    type: 'PreRuntime',
    index: 6,
    value: Type(2) [
      [Type],
      [Bytes [Uint8Array]],
      registry: TypeRegistry {},
      _Types: [Array]
    ]
  },
  {
    type: 'Seal',
    index: 5,
    value: Type(2) [
      [Type],
      [Bytes [Uint8Array]],
      registry: TypeRegistry {},
      _Types: [Array]
    ]
  }
]
```